index.js

```diff
@@ -2,10 +2,11 @@ var bugsnag = require('bugsnag'),
     util = require('util'),
     winston = require('winston')
 
-function Bugsnag () {
-
-  this.name = 'bugsnag'
+function Bugsnag (options) {
+  options = options || {};
 
+  this.name = 'bugsnag';
+  this.level = options.level || 'silly';
```

Comment:
i'm not sure I understand what this does. the logger already supports levels - is this setting the *maximum* level?